.github/dependabot.yml

```diff
@@ -14,3 +14,107 @@ updates:
       include: "scope"
     ignore:
       - dependency-name: "gaurav-nelson/github-action-markdown-link-check"
+
+  - package-ecosystem: "docker"
+    directory: "/components/operator"
+    labels:
+      - "area/dependency"
+      - "kind/chore"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "operator"
+      include: "scope"
+
+  - package-ecosystem: "docker"
+    directory: "/components/runtimes/java17"
+    labels:
+      - "area/dependency"
+      - "kind/chore"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "java17"
+      include: "scope"
+  - package-ecosystem: "docker"
+    directory: "/components/runtimes/nodejs/nodejs16"
+    labels:
+      - "area/dependency"
+      - "kind/chore"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "nodejs16"
+      include: "scope"
+  - package-ecosystem: "docker"
+    directory: "/components/runtimes/nodejs/nodejs18"
+    labels:
+      - "area/dependency"
+      - "kind/chore"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "nodejs18"
+      include: "scope"
+  - package-ecosystem: "docker"
+    directory: "/components/runtimes/python39"
+    labels:
+      - "area/dependency"
+      - "kind/chore"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "python39"
+      include: "scope"
+
+  - package-ecosystem: "docker"
+    directory: "/components/serverless/deploy/jobinit"
```

Comment:
Images pinned to SHA; pinning `europe-docker.pkg.dev/kyma-project/prod/external/*` is not necessary, as the implementation of the syncer guarantees the synced tags are immutable